### DISCUSSION TITLE
Please release an update with new CsWinRT update

### DISCUSSION THREAD

**Discussion by Marv51** [microsoft/WindowsAppSDK#1436] [Q&A] [answered]
Hi, 
Could you please release an update for 0.8 that was built using the new 1.3.5 release of CsWinRT?
For me the regression in Polygon (microsoft/cswinrt#971) is really a big issue. This regression was fixed along with multiple other bugs in this latest update. The Polygon bug has also been reported multiple times in the WinUI repo by different people.

Thank you for considering this.

**Comment by riverar** [comment, marked as answer]
@andrewleader @dayman40 Servicing candidate

**Reply by btueffers** [reply]
Thanks @Marv51! We're planning to update CSWinRT in a future 0.8 servicing release.

**Reply by Marv51** [reply]
That's great to hear. 

I do have to admit that I was a bit frustrated that a new update was released earlier today without this fix.

**Reply by andrewleader** [reply]
Thanks @Marv51, we understand it might have been frustrating to see 0.8.4 be released without the updated CsWinRT version, but we were working on the 0.8.4 servicing update much earlier. Updating versions of CsWinRT always adds some risk and requires more time (sometimes we encounter unexpected issues), and we had some critical fixes we needed to get out with 0.8.4 first.

**Comment by Marv51** [comment]
While this discussion was marked as answered. The issue has not been solved, as no update for 0.8 containing the updated CsWinRT version has been released. 

The timeline for fixing and releasing a regression introduced in a servicing update is honestly a bit disappointing.
